### PR TITLE
Copy script to pull and setup repos in VM

### PR DESCRIPTION
**Pull request by sauloperez**
This copies the setup_repos.sh in the HOME as part of the provisioning, then you need to ssh into the machine and execute it.

It pulls both katuma and katuma-web repos and install their dependencies, including
setting up the database with its seed data.

This makes trivial setting up the environment and so easier for others to join the project. We can remove steps from the wiki! 🎉 

- [ ] Update the wiki once merged.

### CODE REVIEW

**Review by enricostano — CHANGES_REQUESTED** [review]
I think it would be better to actually copy the script to the VM/container from Ansible, no need here to use more custom shell provisioning.

EDIT: Probably the best approach should be to create a new `development` role since this would be only needed for `development.yml`.

**Comment by sauloperez** [comment]
Agree

File: setup_repos.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+cd $HOME
+git clone git@github.com:coopdevs/katuma.git
```
Comment (enricostano) [inline setup_repos.sh:4]: The git clone is not needed for `LXC` containers.

File: setup_repos.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+cd katuma
+bundle install
+bundle exec rake db:setup
+
+cd $HOME
+git clone git@github.com:coopdevs/katuma-web.git
```
Comment (enricostano) [inline setup_repos.sh:10]: ditto

**Review by enricostano — CHANGES_REQUESTED** [review]

**Comment by sauloperez** [comment]
I'll reopen it doing it as @enricostano suggested

**Pull request closed without merging**